packages/okta-angular/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 [#776](https://github.com/okta/okta-oidc-js/pull/776) `OktaAuthGuard` now implements `canActivateChild`
 
+### Bug Fixes
+
+[#794](https://github.com/okta/okta-oidc-js/pull/794) `OktaAuthService.getUser` Only use `/userinfo` endpoint to retrieve user claims.
```

Comment:
We should expand this to include a reference about what to do if they were depending on the idToken.claims option.